components/Contrib.js

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { useRouter } from 'next/router';
+import { Container, Nav, Navbar, Table } from 'react-bootstrap';
+import TimeAgo from 'react-timeago';
+import { AlertIcon, LinkIcon } from '@primer/octicons-react';
+import { dateSort, numericSort, sortData } from 'lib/utils/sort';
+import YesNoBool from 'components/YesNoBool';
+import HeaderLink from 'components/HeaderLink';
+import ActiveLink from 'components/ActiveLink';
+
+// These views display assignment columns.
+const sortConfig = {
+  priority: {},
+  title: {},
+  repo: {},
+  assigned: {
+    sortFunc: numericSort,
+  },
+  mentorAssigned: {
+    sortFunc: numericSort,
+  },
+  updatedAt: {
+    sortFunc: dateSort,
+  },
+};
+
+function renderRows({ data, hasAssignments }) {
+  const rows = [];
+  const colSpan = 6;
+
+  if (!data) {
+    return (
+      <tr>
+        <td colSpan={colSpan}>Loading...</td>
+      </tr>
+    );
+  }
+
+  if (data.length === 0) {
+    return (
+      <tr>
+        <td colSpan={colSpan}>
+          <div className="not-found">
+            <p>
+              No issues found! Time to deploy the team to find some quality
+              bugs!
+            </p>
+          </div>
+        </td>
+      </tr>
+    );
+  }
+
+  for (let i = 0; i < data.length; i++) {
+    const issue = data[i];
+
+    rows.push(
+      <tr key={`issue-${i}`}>
+        <td>
+          <span className={issue.priority || 'unprioritized'}>
+            {issue.priority ? issue.priority.toUpperCase() : <AlertIcon />}
+          </span>
+        </td>
+        <td>
+          <a href={issue.url} target="_blank" rel="noopener noreferrer">
+            <strong>#{issue.number}:</strong> {issue.title}{' '}
+            <LinkIcon verticalAlign="middle" />
+          </a>
+        </td>
+        <td>{issue.repository.name.replace('addons-', '')}</td>
+        {hasAssignments ? (
+          <td className="centered">
+            <YesNoBool
+              bool={issue.assigned}
+              extraClasses={{
+                yes: ['contributor'],
+                no: ['contributor', 'not-assigned'],
+              }}
+            />
+          </td>
+        ) : null}
+        {hasAssignments ? (
+          <td className="centered">
+            <YesNoBool
+              bool={issue.mentorAssigned}
+              extraClasses={{
+                yes: ['mentor'],
+                no: ['mentor', 'not-assigned'],
+              }}
+            />
+          </td>
+        ) : null}
+        <td>
+          <TimeAgo date={issue.updatedAt} />
+        </td>
+      </tr>,
+    );
+  }
+  return rows;
+}
+
+const Contrib = (props) => {
+  const router = useRouter();
+  const { dir, sort } = router.query;
+  const { contribData, hasAssignments } = props;
+
+  let data = contribData;
+  if (sort) {
+    data = sortData({ data, columnKey: sort, direction: dir, sortConfig });
+  }
+
+  return (
+    <div className="Contrib">
+      <Helmet>
```

Comment:
I think you can just use next's built-in `Head` for this, or do you need to use `Helmet` everywhere because you need to use it somewhere?

Reply:
Unfortunately I need to use react-helmet to set classes on the body for the dark background on the dashboard. Without this the only way to do it is with useEffect, but that's not fast enough to not see a flash when it's added when navigating to the dashboard page.